### PR TITLE
Lecture 3 problems

### PR DESCRIPTION
**Pull request by M-Ahmed-Khan**
Problem-1: Take two numbers as input and add them
Problem-2: Take a number as input and determine whether it is even or odd
Problem-3: Take a number and print it 10 times

### CODE REVIEW

File: problem-03.c [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+int main()
+{
+    int number;
+    printf("Enter a number: ");
+    scanf("%d", &number);
+
+    for (int i = 0; i < 10; i++)
+    {
+        printf("\n%d", number);
```
Comment (nf-khan) [inline problem-03.c]: Why are you adding newline character before number? I think it should be added after the number.

**Review by nf-khan — CHANGES_REQUESTED** [review]
1. Provide a meaningful description of PR (implement this in all PRs).
2. I think you should comment task description in line # 2 in each program.

File: problem-03.c (hunk 1 above)
Comment (M-Ahmed-Khan) [reply problem-03.c]: Updated PR according to your comment

**Review by nf-khan — APPROVED** [review]
Approved

**Pull request merged**